app/styles/columns.less

```diff
@@ -74,6 +74,11 @@
     white-space: nowrap;
     text-overflow: ellipsis;
     overflow: hidden;
+    color: darken(@upf-gray-light, 40%);
+    font-size: 15px;
```

Comment:
use the font sizes that we have with the `/@rem` (it might not be perfect but it's just going to be worse if we put whatever font-sizes we want)

Reply:
it's always possible to get a closest match to 15px